### PR TITLE
Add missing ooxml-schemas dependency #8081

### PR DESCRIPTION
**Pull request by galderz**
~This is a draft PR~.

Required to avoid [issue](https://github.com/quarkusio/quarkus/issues/8081#issuecomment-604014138).

Note that this change happens to be part of the bigger #7198 PR. I came to it separately following the advice [here](https://stackoverflow.com/questions/49483042/apache-poi-java-lang-noclassdeffounderror). Talking to @sberyozkin he pointed out it was part of the other PR.

### CODE REVIEW

**Comment by galderz** [comment]
Ups, too many commits there, let me revert the others.

**Comment by galderz** [comment]
Oh crap, messed up commits, one sec...

**Comment by galderz** [comment]
Fixed now 😅

File: extensions/tika/runtime/pom.xml [hunk 1]
```diff
@@ -43,6 +43,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```
Comment (sberyozkin) [inline extensions/tika/runtime/pom.xml]: @galderz Thanks for this PR. My only reservation is that with `quarkus-tika`, if a user configures a PDF parser only for example (`quarkus.tika.parser=pdf`)), then so far none of the non-PDF dependencies are expected to make it to the native image. 
I wonder, if this dependency were optional, then would making a build step which would check if `OOXML` is enabled (can be true by default to start with - I can add an `ooxml` shortcut later) and if yes - add a native resource build item which would make this `ooxml-schemas` included ?
Is it realistic ?

Comment (galderz) [reply extensions/tika/runtime/pom.xml]: This dependency is only needed during building of the image itself. It's only for class resolution while trying to discover which methods the class has, nothing else. None of the classes that were missing would actually make it to the final image.

Comment (galderz) [reply extensions/tika/runtime/pom.xml]: IOW, it has 0 impact on the pointsto analysis that svm does. So, unless the end application uses something there indirectly, nothing will be included in the native image.

Comment (sberyozkin) [reply extensions/tika/runtime/pom.xml]: Hi @galderz, so for example, `org/openxmlformats/schemas/wordprocessingml/x2006/main/STDisplacedByCustomXml` is missing, and few more I guess, so adding  this dependency also contributes to resolving #7198 (see for example https://github.com/quarkusio/quarkus/pull/7198/files#diff-cefda81cecd3360b76139d0044ecb53aR25). So unless `quarkus-tika` processor references these classes directly or indirectly, no native image size will be affected ?

Comment (sberyozkin) [reply extensions/tika/runtime/pom.xml]: @galderz cool, your last comment makes it quite clear, cheers

**Review by sberyozkin — APPROVED** [review]

**Comment by galderz** [comment]
This is no longer in draft. The change had a positive effect.

**Comment by sberyozkin** [comment]
@galderz Something failed in the native build but I'm pretty sure it is not related, have restarted the build...

**Pull request merged**